### ISSUE TITLE
App Dir & Tailwind SoC questions

### ISSUE THREAD

**Issue by appjitsu** [Blazity/next-enterprise#32]
Great project! A lot of work went into this.

- Why not use the new app directory?
- Why not use a tailwind ui library like Chakra? The problem with base tailwind is the tsx can be too crowded with tailwind classes.

**Comment by appjitsu** [comment]
Understood. Thanks for the feedback.

**Comment by 0xRowdy** [comment]
One more question?
Why did you decide to stick with yarn instead of pnpm?

**Comment by bmstefanski** [comment]
@0xRowdy 
> One more question.
Why did you decide to stick with yarn instead of pnpm?

Yarn is battle-tested and still fast (not as fast as pnpm, but good enough), its main drawback is that it takes a lot of disk space, but that's not a worry for most projects. I personally like pnpm and use it in smaller/personal projects, but using it with mono repo tools like Turborepo or lerna can really be hard or even impossible sometimes because of its immaturity.

So to sum it up: having a mature package manager that will work with every library and tool was more important than speed benefits.

**Comment by rharkor** [comment]
> Great project! A lot of work went into this.
> 
> * Why not use the new app directory?
> * Why not use a tailwind ui library like Chakra? The problem with base tailwind is the tsx can be too crowded with tailwind classes.

Hy @appjitsu , I've forked this project and included a lot of cool features like the app directory, shadcn components, authentication and more. If you want to give a look https://github.com/rharkor/next-boilerplate

**Comment by bmstefanski** [comment]
Updated to app directory, closing.
